toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -237,3 +237,5 @@ python3-libs-3.7.7-10.cm1.aarch64.rpm
 python3-setuptools-3.7.7-10.cm1.noarch.rpm
 python3-xml-3.7.7-10.cm1.aarch64.rpm
 systemd-rpm-macros-239-37.cm1.noarch.rpm
+which-2.21-7.cm1.aarch64.rpm
```

Comment:
Please also update [this line inside update_manifests.sh](https://github.com/microsoft/CBL-Mariner/blob/769d5cb4af430c99a3ef99b514e899aa9766ccdd/toolkit/resources/manifests/package/update_manifests.sh#L200) - instead of removing `which` I think we'd only need to remove `which-debuginfo`.

Reply:
Updated